lib/inspec/schema/profile_json.rb

```diff
@@ -0,0 +1,58 @@
+require_relative "./primitives"
+
+# We disable this for the sake of legibility
+# rubocop:disable Layout/AlignHash
```

Comment:
We should avoid adding new rubocop:disables to the code.

Reply:
Done. Having them aligned was nice for working, but unnecessary for dev.